mne/io/egi/tests/test_egi.py

```diff
@@ -212,7 +212,7 @@ def test_io_egi_mff(events_as_annotations):
     if events_as_annotations:
         # Grab the first annotation. Should be the first "DIN1" event.
         assert len(raw.annotations)
-        onset, dur, desc, _ = raw.annotations[0].values()
+        onset, dur, desc, _, _ = raw.annotations[0].values()
```

Comment:
this tells me it's a breaking change but I doubt this will affect many people...